kernel/src/main/java/org/kframework/kbmc/KBMCOptions.java

```diff
@@ -0,0 +1,58 @@
+package org.kframework.kbmc;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParametersDelegate;
+import org.kframework.main.GlobalOptions;
+import org.kframework.unparser.PrintOptions;
+import org.kframework.utils.errorsystem.KEMException;
+import org.kframework.utils.file.FileUtil;
+import org.kframework.utils.inject.RequestScoped;
+import org.kframework.utils.options.DefinitionLoadingOptions;
+import org.kframework.utils.options.SMTOptions;
+
+import java.io.File;
+
+@RequestScoped
+public class KBMCOptions {
+
+    @ParametersDelegate
+    public transient GlobalOptions global = new GlobalOptions();
+
+    @ParametersDelegate
+    public DefinitionLoadingOptions definitionLoading = new DefinitionLoadingOptions();
+
+    @Parameter(names={"--spec-file"}, description="Path to the specification file.")
```

Comment:
let's call this --raw-spec and give it a description like "Path to a file containing the patterns to model-check. These patterns will be executed as-is without inserting any initial configuration."

Reply:
done